macros/network/lb-create-backend-4-healthcheck.mdx

```diff
@@ -0,0 +1,17 @@
+---
+macro: lb-create-backend-4-healthcheck
+---
+
+<Lightbox src="scaleway-configure-health-checks.webp" alt="" />
+
+Health checks are regular, automized attempts by the Load Balancer to communicate with backend servers. This ensures that servers are listening and healthy.
+
+1. Select a health check **type**. Various protocols are available, including HTTP, TCP, MYSQL, and more. Depending on the health check type you choose, you may be prompted to enter additional information.
+
+2. Choose whether to use **proxy protocol** during health checks. Proxy protocol enables the original client IP address to be passed to the backend server in a standardized way during health checks. The proxy protocol setting for health checks is inherited from the backend configuration but you can choose to override it. For more information, click on the link in the green tip box below.
+
+3. Choose whether to edit more settings for health checks by clicking the **Advanced settings** button. Otherwise, these settings will be left at the recommend default values. 
```

Comment:
```suggestion
3. Choose whether to edit more settings for health checks by clicking the **Advanced settings** button. Otherwise, these settings will be left at the recommended default values. 
```